hi-ml/src/health_ml/utils/data_augmentations.py

```diff
@@ -37,45 +37,44 @@ def __init__(self, theta: float = 0.) -> None:
         self.hed_from_rgb = torch.tensor([[1.87798274, -1.00767869, -0.55611582],
                                           [-0.06590806, 1.13473037, -0.1355218],
                                           [-0.60190736, -0.48041419, 1.57358807]])
+        self.log_adjust = torch.log(torch.tensor(1E-6))
 
-    @staticmethod
-    def adjust_hed(img: torch.Tensor,
-                   theta: float,
-                   stain_from_rgb_mat: torch.Tensor,
-                   rgb_from_stain_mat: torch.Tensor
-                   ) -> torch.Tensor:
+    def adjust_hed(self, img: torch.Tensor) -> torch.Tensor:
         """
         Applies HED jitter to image.
 
         :param img: Input image.
-        :param theta: Strength of the jitter. HED_light: theta=0.05; HED_strong: theta=0.2.
-        :param stain_from_rgb_mat: Transformation matrix from HED to RGB.
-        :param rgb_from_stain_mat: Transformation matrix from RGB to HED.
         """
-        alpha = torch.FloatTensor(1, 3).uniform_(1 - theta, 1 + theta)
-        beta = torch.FloatTensor(1, 3).uniform_(-theta, theta)
 
-        # Only perturb the H (=0) and E (=1) channels
-        alpha[0][-1] = 1.
-        beta[0][-1] = 0.
+        alpha = torch.FloatTensor(img.shape[0], 3).uniform_(1 - self.theta, 1 + self.theta)
+        beta = torch.FloatTensor(img.shape[0], 3).uniform_(-self.theta, self.theta)
```

Comment:
Why not already instantiate `alpha` and `beta` with shape `(batch, 1, 1, 3)`, if below you do `[:, None, None, :]` anyway?